README.md

```diff
@@ -2,4 +2,12 @@
 A set of scripts to access, analyse image and metadata from OMERO.
 
 Under ``practical`` are the files that can be used during a training session.
+The scripts under ``practical`` are no longer maintained.
+They have been moved to various repositories:
+ - The files under ``practical/groovy`` have been moved to [omero-guide-fiji](https://github.com/ome/omero-guide-fiji/tree/master/scripts).
+ - The files under ``practical/javascript`` have been moved to [omero-guide-figure](https://github.com/ome/omero-guide-figure/tree/master/scripts).
+ - The files under ``practical/matlab`` have been moved to [omero-guide-matlab](https://github.com/ome/omero-guide-matlab/tree/master/scripts).
+ - The files under ``practical/orbit`` have been moved to [omero-guide-orbit](https://github.com/ome/omero-guide-orbit/tree/master/scripts).
+  - The files under ``practical/python`` have been moved to [omero-guide-python](https://github.com/ome/omero-guide-python/tree/master/scripts).
```

Comment:
```suggestion
 - The files under ``practical/python`` have been moved to [omero-guide-python](https://github.com/ome/omero-guide-python/tree/master/scripts).
```

Not sure how many spaces should be from the left, but the last line is shifted here - although it still builds well in display